src/main/kotlin/org/rust/lang/core/resolve/NameResolution.kt

```diff
@@ -706,8 +706,8 @@ private fun processAssociatedItemsWithSelfSubst(
     selfSubst: Substitution,
     processor: RsResolveProcessor
 ): Boolean {
-    val assocItemsProcessor: RsResolveProcessor = {
-        processor(it.name, it.element!!, it.subst + selfSubst)
+    val assocItemsProcessor: (AssocItemScopeEntry) -> Boolean = {
```

Comment:
I found 3 usages of `(AssocItemScopeEntry) -> Boolean` type. Maybe create alias for it?

Reply:
I did it without mentioning this type =)